src/query.ts

```diff
@@ -245,6 +251,11 @@ export function stringifyEncodingQueryFieldDef(encQ: EncodingQuery): string {
     if (encQ.bin['maxbins']) {
       params.push({key: 'maxbins', value: encQ.bin['maxbins']});
     }
+  } else if (encQ.scale && !isEnumSpec(encQ.scale)) {
```

Comment:
encQ.scale should not be a part of this `if-else` clause.

scale is orthogonal to function (aggregate, timeUnit, bin).

Also a test to make sure the scale type + one of the combo works.   (I also commented about this the test file.)